docs/pages/props/index.tsx

```diff
@@ -93,17 +88,9 @@ export default function Api() {
     These base props are those available to be passed to all select variants.
 
     ${(
-      <Props
-        props={require('!!extract-react-types-loader!../../PropTypes/Select')}
-        overrides={{
-          components: (props) => (
-            <Prop
-              {...props}
-              shapeComponent={() => null}
-              type="All Components Object"
-            />
-          ),
-        }}
+      <ShowTypes
+        getNode={getNode}
+        index={selectTypes['SelectInstance'].index}
```

Comment:
This should be the select props, right? it looks like it's displaying the select instance though? (and same for the one above)

Reply:
yep, good catch, changed both